src/Features/Core/Portable/UseCoalesceExpression/AbstractUseCoalesceExpressionDiagnosticAnalyzer.cs

```diff
@@ -1,13 +1,15 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.CodeStyle;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.LanguageServices;
 
 namespace Microsoft.CodeAnalysis.UseCoalesceExpression
 {
+    /// <summary>
+    /// Looks for code of the form "x == null ? y : x" and offers to convert it to "x ?? y";
```

Comment:
and null == x, and null != x;  and (x == null) and (x) == null. :)

I'm just trying to provide a comment htat explains the general purpose of the analyzer.  An exhaustive list would be as long as all the recognition code.